### DISCUSSION TITLE
Διαγραφή παλιού φόρκ μαθήματος και δημιουργία νέου

### DISCUSSION THREAD

**Discussion by NestorasKosmidis** [courses-ionio/sw#1235] [Q&A] [answered]
Μπορούμε να επιλέξουμε για το 2ο παραδοτέο όποια άσκηση θέλουμε ενώ για το 6ο αποκλειστικά αυτό για τα blog ;

**Comment by epidrome** [comment, marked as answer]
@NestorasKosmidis καλύτερα να διαγράψεις το παλιό φορκ του μαθήματος και να κάνεις νέο, γιατί οι ασκήσεις είναι λίγο διαφορετικές φέτος
